src/code/z_play_hireso.c

```diff
@@ -16,12 +16,12 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016EA90.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016F1A8.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/Hireso_Draw.s")
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016F4EC.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016F5A8.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/Hireso_Update.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016FC78.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/Hireso_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016FC98.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/Hireso_Destroy.s")
```

Comment:
Since you're working on Hireso, I wonder if we could consider using `Notebook_` prefixes for functions? So:
`Notebook_Init`
`Notebook_Update`
...
Maybe something to consider